package.json

```diff
@@ -24,22 +24,22 @@
     "test:watch": "nodemon --exec jasmine JASMINE_CONFIG_PATH=jasmine.json"
   },
   "dependencies": {
-    "@adobe/jwt-auth": "^0.3.0",
+    "@adobe/jwt-auth": "^0.3.3",
     "chalk": "^4.1.0",
-    "delay": "^4.2.0",
-    "inquirer": "^7.3.2",
-    "ora": "^4.0.4",
+    "delay": "^4.4.0",
+    "inquirer": "^7.3.3",
+    "ora": "^5.3.0",
```

Comment:
`5.x` "breaking change" was requiring `node 10`, which we already had in `package.json` of this repo.